geps/gep-1619.md

```diff
@@ -106,10 +200,6 @@ The following are items that we intend to resolve before we consider this GEP im
 - We need to add a small section on compliance regarding the browser and client relationship.
 - We need to design and document an API for session persistence and/or session affinity in the Gateway API spec.
 
-## API
```

Comment:
For line 198 (session draining) please check out https://github.com/grpc/proposal/blob/master/A55-xds-stateful-session-affinity.md#background (last para).

Reply:
For line 197 (needs and use-cases) check out https://static.sched.com/hosted_files/kccnceu2023/c4/Autoscaling%20Elastic%20Kubernetes%20Infrastructure%20for%20Stateful%20Applications%20using%20Proxyless%20gRPC%20and%20Istio.pdf (slide 17 onwards).

Reply:
Ack - added both links as references, but I'll keep this in mind as I work on this GEP.